README.md

```diff
@@ -64,6 +66,72 @@ dsc_windowsfeature {'IIS':
 
 All DSC Resource names and parameters have to be in lowercase, e.g: `dsc_windowsfeature` or `dsc_name`.
 
+### Setting Registry Values
+
+Creating and modifying Registry keys and values is easy with Puppet and DSC.
```

Comment:
and easier if you just use Puppet to do so. I'm not sure if this is a good example to use.